### PR TITLE
bugfix: ignore commented-out schema tests

### PR DESCRIPTION
**Pull request by cmcarthur**
Fix for #328

### CODE REVIEW

File: dbt/parser.py [hunk 1]
```diff
@@ -256,6 +256,9 @@ def parse_schema_tests(tests, root_project, projects):
 
         for model_name, test_spec in test_yml.items():
             for test_type, configs in test_spec.get('constraints', {}).items():
```
Comment (drewbanin) [inline dbt/parser.py]: can we also check if `test_spec is None`? That would happen here:
```yml
test_model:
    constraints:
        # unique:
        #- some_field
```

Edge case, but would be good to avoid the ol stack trace :)

Comment (drewbanin) [inline dbt/parser.py]: Would also be good to check that `test is not None` on line 253 here:

```yml
test_model:
#    constraints:
#        unique:
#            - a
```

**Comment by cmcarthur** [comment]
@drewbanin I added some more edge case tests and put in the fixes. See what I did for test==None, I don't fully understand your suggestion but I think `test_yml is None` might cover it

**Comment by drewbanin** [comment]
lgtm
